index.html

```diff
@@ -212,6 +211,39 @@ <h1 class="about-section-h1">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <section class="form-section">
+            <h2 class="form-h2">I'm always<br>interested in hearing<br>about new projects,<br>so if you'd like to<br>chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xoqypwnp" method="post">
+                    <ul class="ul-for-form">
+                        <li>
+                            <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input type="email" name="user_email" id="mail" placeholder="Email Address" required>
+                        </li>
+                        <li class="txt"> 
+                            <textarea  class="text-input" name="user_message" id="msg" cols="10" rows="5" placeholder="Enter text here" maxlength="500" required></textarea>
+                        </li>
+                        <li class="button-container">
+                            <input class="get-button" type="submit" name="submit" value="Get in touch">
```

Comment:
- [ ] **OPTIONAL**For that like submissions I strongly recommend to use `<button>` tag, so your code should be semantic as much as you can.